### PR TITLE
Uxods 383  remove aria multiselectable from accordion

### PR DESCRIPTION
**Pull request by wc0599**
**List of Major Changes**
1. Migrate accordion related styles & functionality from Design-System to Design-System-Angular.
2. Alter Accordion Docsite to reflect changes.
3. Add Accordion Storybook tabs to test--Single-select, Multi-select, Bordered, and Accordion with Interactive HTML components.
4. Add Accordion Jest unit tests.

### CODE REVIEW

File: docs/components/accordion/accordion-bordered-example.code [hunk 1]
```diff
@@ -1,8 +1,8 @@
-<jazz-accordion [showBorder]="true">
+<jazz-accordion [showBorder]="true" [multiSelect]="true">
```
Comment (rs16princeton) [inline docs/components/accordion/accordion-bordered-example.code:1]: Default examples should be multi-select = false, when multiselect is not specified it should default to false as well

File: docs/components/accordion/accordion-borderless-example.code [hunk 2]
```diff
@@ -1,8 +1,8 @@
-<jazz-accordion [showBorder]="false">
+<jazz-accordion [showBorder]="false" [multiSelect]="true">
```
Comment (rs16princeton) [inline docs/components/accordion/accordion-borderless-example.code:1]: same here as above

File: docs/components/accordion/accordion-example.code [hunk 3]
```diff
@@ -1,8 +1,8 @@
-<jazz-accordion>
+<jazz-accordion [multiSelect]='true'>
```
Comment (rs16princeton) [inline docs/components/accordion/accordion-example.code:1]: and here

File: projects/design-system-angular/src/lib/accordion/accordion.component.spec.ts [hunk 4]
```diff
@@ -1,25 +1,389 @@
...
+import {
+  ACCORDION_CONTENT_EXPANDED_CLASSNAME,
+  ACCORDION_MULTISELECTABLE_CLASSNAME,
+  AccordionComponent
+} from './accordion.component';
+import {Component, ViewChild} from "@angular/core";
+
+@Component({
+  template: `
+    <jazz-accordion [multiSelect]="false">
```
Comment (rs16princeton) [inline projects/design-system-angular/src/lib/accordion/accordion.component.spec.ts:12]: make sure your tests are testing without the multiSelect attribute, we want to make sure the accordions work even without the attribute specified

File: docs/components/accordion/accordion-bordered-example.code (hunk 1 above)
Comment (wc0599) [reply docs/components/accordion/accordion-bordered-example.code:1]: The current behavior in master branch for Design-System project has single-select as default, whereas Design-System-Angular has multi-select as default without markers for "multi-select".

As discussed, I will change the default behavior for multi-select accordions in Design-System-Angular to match that of Design-System so that when "multi-select" is not specified, it would default to "single-select".

File: projects/design-system-angular/src/lib/accordion/accordion.component.ts [hunk 5]
```diff
@@ -9,40 +9,42 @@ import {
...
 export const ARIA_CONTROLS = 'aria-controls';
-export const HIDDEN = 'hidden';
+export const HIDDEN        = 'hidden';
 
 /**
  * An Accordion is a vertically stacked set of headings that each control the visibility of an associated content section.
  *
  * <example-url>http://localhost:4200/jazz-design-system/#/accordion/accordionExample</example-url>
  * @example
-   `` `
```
Comment (rs16princeton) [inline projects/design-system-angular/src/lib/accordion/accordion.component.ts:23]: Why did you remove the indentation, this makes it harder to read if you remove it.

Comment (wc0599) [reply projects/design-system-angular/src/lib/accordion/accordion.component.ts:23]: For an unknown default setting, my Intellij auto-formats spacings on save for some things. I'll add the spacings back in.

File: projects/design-system-angular/src/lib/accordion/accordion.component.html [hunk 6]
```diff
@@ -1,5 +1,6 @@
-<div class="jazz-accordion {{showBorder?'jazz-accordion--bordered':''}}"
-     [attr.aria-multiselectable]="multiSelect ? true : false"
+<div class="jazz-accordion
```
Comment (rs16princeton) [inline projects/design-system-angular/src/lib/accordion/accordion.component.html:1]: Can you try setting the classes by using ngClass? This is typically how dynamic classes are set in angular and it is more explicit for the next developer to understand:
https://www.angularjswiki.com/angular/how-to-add-a-class-based-on-condition-in-angular/
https://www.developer.com/languages/javascript/assigning-dynamic-classes-with-ngclass/

You can delimit the classes with a comma if you need to do multiple classes.

**Review by rs16princeton — APPROVED** [review]
